Admin/AdminHelper.php

```diff
@@ -32,25 +28,6 @@ public function __construct(Pool $pool)
     }
 
     /**
-     * @throws \RuntimeException
-     *
-     * @param \Symfony\Component\Form\FormBuilder $formBuilder
-     * @param string                              $elementId
-     *
-     * @return \Symfony\Component\Form\FormBuilder
-     */
-    public function getChildFormBuilder(FormBuilder $formBuilder, $elementId)
```

Comment:
Removing a public method is completely BC break.

BTW, why removing it?

If it's necessary, please make BC with deprecation system.